real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/messages/IMUCompassMessage.java

```diff
@@ -0,0 +1,40 @@
+package com.roboclub.robobuggy.messages;
+
+import java.util.Date;
+
+/**
```

Comment:
In reference to the last commit, are the other messages not going to use this @author, @version, changelog, description structure?